src/main/java/core/utils/date/DateManager.java

```diff
@@ -82,4 +82,16 @@ private void convertWordToDate(final String wordDate) {
                 throw new InvalidArgumentException("Invalid Argument: Unsupported String Format.");
         }
     }
+
+    /**
+     * Generates actual date.
+     *
+     * @param dataFormat is format of data
+     * @return a string with actual date
+     */
+    public static String generateDateActual(final String dataFormat) {
+        java.text.DateFormat dateFormat = new SimpleDateFormat(dataFormat);
```

Comment:
Shouldn't have been better import the Dateformar instead of this declaration? or what advantages we get with this type of declaration?

Reply:
I will fix that